pkg/build/fuchsia.go

```diff
@@ -92,3 +104,21 @@ func runSandboxed(timeout time.Duration, dir, command string, arg ...string) ([]
 	}
 	return osutil.Run(timeout, cmd)
 }
+
+func genSSHKeys(dir string) (string, string, error) {
```

Comment:
I'd suggest adding a documenting comment (regarding parameters/return values, as well as the fact that this won't regenerate if they already exist), and consider using named return values so it's easier to see which string is which in the signature.